_data/ph_authors.yml

```diff
@@ -1357,3 +1357,12 @@
     - editorial
     - proghist
   status: volunteer
+
+- name: Josir Cardoso Gomes
+  team: false
+  orcid: 0000-0001-7629-1404
+  bio:
+      en: |
```

Comment:
The Spanish translation is missing. I'll add it tomorrow. Whenever you need a translation, you can tag the team + the Managing Editor to request it (https://github.com/programminghistorian/jekyll/wiki/Requesting-Translation-Guidelines)